docs/registering-a-control-handler-function.md

```diff
@@ -25,12 +27,13 @@ When a **CTRL\_CLOSE\_EVENT** signal is received, the control handler returns **
 
 When a **CTRL\_BREAK\_EVENT**, **CTRL\_LOGOFF\_EVENT**, or **CTRL\_SHUTDOWN\_EVENT** signal is received, the control handler returns **FALSE**. Doing this causes the signal to be passed to the next control handler function. If no other control handlers have been registered or none of the registered handlers returns **TRUE**, the default handler will be used, resulting in the process being terminated.
 
+> [!NOTE]
+> Calling [**AttachConsole**](attachconsole.md), [**AllocConsole**](allocconsole.md), or [**FreeConsole**](freeconsole.md) will reset the table of control handlers in the client process to its initial default state. Handlers must be registered again when the attached console session changes.
```

Comment:
```suggestion
> Calling [**AttachConsole**](attachconsole.md), [**AllocConsole**](allocconsole.md), or [**FreeConsole**](freeconsole.md) will reset the table of control handlers in the client process to their initial default state. Handlers must be registered again when the attached console session changes.
```

Reply:
Their for "control handlers" instead of its for "table" is what you're saying?

Reply:
Yeah. Because control handlers is plural I assumed that it needs to be "their" initial states, since there are multiple. Is that incorrect?

Reply:
Depends how pedantic you want to be about English. The way I was taught, "table" is the object that is being "reset" therefore it should be "its" as if the additional prepositional phrase "of control handlers" describing what's in the table did not exist in this sentence. This one's done both ways all the time, though, so whichever is clearer to readers is fine with me.